cinemy/cinema/tasks/tasks.py

```diff
@@ -0,0 +1,25 @@
+from django.utils import timezone
+from cinema.models import PlayingTime
+from django.db.models import F
+from datetime import timedelta
+
+
+def clean_unconfirmed_reservations():
+    """
+    This function will clean all the unconfirmed reservations
+    at 30 minutes prior to starting the movie
+    """
+    time_now = timezone.now()
+    playing_times = PlayingTime.objects.annotate(
+        time_min=(time_now - F("start_time"))
+    ).filter(time_min__lte=timedelta(minutes=30))
+    for playing_time in playing_times:
```

Comment:
I wrote on the group, modified the ticket. Please modify this, we only need to mark them as expired, not delete them.

Reply:
Done